doc/nrf/known_issues.rst

```diff
@@ -132,21 +133,21 @@ CIA-604: ATv2 cannot be built for the ``thingy91_nrf9160_ns`` build target with
 Serial LTE Modem
 ================
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
   Building the application for Thingy:91 fails if secure_bootloader overlay is included.
 
 Other issues
 ============
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
```

Comment:
```suggestion
.. rst-class:: v1-9-1 v1-9-0
```

Reply:
This issue has been resolved as well. CC @tokangas

Reply:
Yes, this issue is valid only with v1.9.0 and v1.9.1.